okhttp/src/test/java/io/grpc/okhttp/OkHttpClientTransportTest.java

```diff
@@ -181,16 +181,6 @@ public long read() {
     }
   }
 
-  /** Final test checks and clean up. */
-  @After
-  public void tearDown() throws Exception {
```

Comment:
It would be possible, but adds additional complexity.  Tests that want a clientTransport (like mine) use this variable rather than needing to declare a new one. 

Having each test do its own verification allows for a very convenient style of test writing where a new test can be copied from one of the existing test.  Effectively each test gains independence from the others in the suite since they don't share common setup and tear down (which, in this case is good, because they really don't!)  Also it means that tests that are no longer relevant can be very easily deleted without having to worry about the rest.

There is also a really big readability benefit from this approach as well.  You can read the test function body _alone_ and figure out what is going on.  There is dramatically less need to look around in the rest of the file, to see  what is happening.  

The only major downside from this is the code duplication, but I think it is forgivable due to the extreme ease of writing more tests.  

Thoughts?